app/api/api/views.py

```diff
@@ -360,7 +360,7 @@ class ScanReportListViewSetV2(ScanReportListViewSet):
     """
 
     filter_backends = [DjangoFilterBackend, OrderingFilter]
-    filterset_fields = {"hidden": ["exact"], "dataset": ["in", "exact"]}
+    filterset_fields = {"hidden": ["exact"], "dataset": ["in", "contains"]}
```

Comment:
```suggestion
    filterset_fields = {"hidden": ["exact"], "dataset": ["in", "icontains"]}
```
`icontains` is the case insensitive version